src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -443,9 +474,133 @@ void getHearingDuration(String benefitCode, String issueCode, String adjournCase
                 .issueCode(issueCode)
                 .adjournCaseNextHearingListingDuration(adjournCaseDuration)
                 .adjournCaseNextHearingListingDurationUnits(adjournCaseDurationUnits)
+                .appeal(Appeal.builder()
+                        .hearingOptions(HearingOptions.builder().build())
+                        .build())
                 .build();
         int result = HearingsDetailsMapping.getHearingDuration(caseData);
 
         assertEquals(expected, result);
     }
+
+    @DisplayName("getHearingDurationBenefitIssueCodes Paper Parameterized Tests")
```

Comment:
Minor: Can we make these display names more descriptive, same as the tests above? Should describe what the test is doing not just tell us what it is.